event_attribution_reporting_clicks.md

```diff
@@ -246,17 +248,6 @@ the browser will compare the priority of the new report with the priorities of t
 If the new report has the lowest priority, it will be ignored. Otherwise, the browser will
 delete the scheduled report with the lowest priority and schedule the new report.
 
-### Multiple sources for the same trigger (Multi-touch)
-
-If multiple sources were clicked and associated with a single attribution trigger, send reports for the one with the highest priority.
-If no priority is specified, the browser performs last-touch.
-
-There are many possible alternatives to this,
```

Comment:
I think we should still keep this section. It's useful to frame that the chosen model: priority based attribution is one possibility, and other models could potentially be considered.